signing/sawtooth_signing/secp256k1.py

```diff
@@ -112,12 +112,13 @@ def sign(self, message, private_key):
         except Exception as e:
             raise SigningError('Unable to sign message: {}'.format(str(e)))
 
-    def verify(self, signature, message, public_key):
+    def verify(self, signature, message, public_key, sig_is_hex=True):
```

Comment:
This is not a very ergonomic way to handle this. More ergonomic alternatives include a new method `verify_bytes()` or doing an `isinstance()` check on signature before processing it: https://docs.python.org/3/library/functions.html#isinstance

I am in favor of the isinstance check, as eventually we will want to move all signatures to bytes. When that happens, we can just remove the check.